integration_test/integration_tests.py

```diff
@@ -184,6 +200,9 @@ def test_integration():
     get_all_templates_for_type(client, EMAIL_TYPE)
     get_all_templates_for_type(client, SMS_TYPE)
 
+    if (os.environ['INBOUND_SMS_QUERY_KEY']):
+        get_received_text_messages()
```

Comment:
i guess this is for when we run locally?

Reply:
yes, otherwise it will fail the test unless I take out the test that we get at least 1 item in the response, but it's probably a good thing to check the array items?